### PR TITLE
Require pre-dowloaded files for controlled access data

### PR DESCRIPTION
**Pull request by pinin4fjords**
Fed the correct parameters by the change in https://github.com/ebi-gene-expression-group/scxa-control-workflow/pull/16, requires use of of pre-downloaded files for controlled access analysis.

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -43,10 +49,10 @@ process download_fastqs {
     time { 5.hour * task.attempt }
     memory { 20.GB * task.attempt }
 
-    errorStrategy { task.attempt<=10 ? 'retry' : 'finish' } 
+    errorStrategy { task.attempt<=10 & task.exitStatus != 4 ? 'retry' : 'finish' }
```
Comment (suhaibMo) [inline main.nf:52]: Same question with droplet exit code 4 ? In Smart-seq it was 2 ?

**Review by suhaibMo — APPROVED** [review]
LGTM

File: main.nf (hunk 1 above)
Comment (pinin4fjords) [reply main.nf:52]: If you look at the process code below, you'll see I've set the exit status of 4 for cases when the files are not found locally, and when data are controlled access (so local files a required). This is specific to this workflow- the smartseq one is different.

**Comment by pinin4fjords** [comment]
Thanks @suhaibMo for review

**Pull request merged**